app_feup/lib/model/entities/lecture.dart

```diff
@@ -57,6 +82,19 @@ class Lecture {
     };
   }
 
+
+  Map<String, dynamic> toHtmlMap() {
```

Comment:
Since the Database being used has changed, there won't be any crashes. 
I could try and do so, but as I've said in the other comment, I think it would be wiser for us to reach a middle ground database design. Storing the starttime in seconds does not seem the best approach. Wdyt?

Reply:
To be honest it seems like a better approach than storing them as non-standard strings

Reply:
Going to resolve this as I'm continuing the same discussion above :arrow_up: